moped-editor/src/queries/project.js

```diff
@@ -551,42 +536,6 @@ export const COMPONENTS_QUERY = gql`
   query GetComponents($projectId: Int) {
```

Comment:
this diff renders oddly, but the gist of it is that i added more columns to `COMPONENTS_QUERY` so that we could deprecate `COMPONENT_DETAILS_QUERY`.

this allows us move data that is shared between project components  (`moped_components` and `moped_subcomponents`) into shared state up the DOM tree, saving quite a few renders. it also kills a graphql query at the expense of modestly increasing the size of another.